### PR TITLE
Adds section 17 compliance requirements

### PR DESCRIPTION
**Pull request by MitchHynes**

### CODE REVIEW

File: docs/SRS-Volere/SRS.tex [hunk 1]
```diff
@@ -348,9 +348,17 @@ \subsection{Cultural Requirements}
 
 \section{Compliance Requirements}
 \subsection{Legal Requirements}
-\lips
+\begin{itemize}
+    \item The application is to strictly abide by all regulations governing
```
Comment (Spitgranger) [inline docs/SRS-Volere/SRS.tex:352]: Requirements should have an identifier associated with them to aid in traceability

File: docs/SRS-Volere/SRS.tex [hunk 2]
```diff
@@ -348,9 +348,17 @@ \subsection{Cultural Requirements}
...
+    the treatment of water and wastewater. This includes, but is not limited 
+    to the \textit{Safe Drinking Water Act} (SDWA), 2002, the
+    \textit{Ontario Water Resources Act} (OWRA), the 
+    \textit{Environmental Protection Act} and the \textit{Clean Water Act}.
```
Comment (kyle4908) [inline docs/SRS-Volere/SRS.tex:356]: These don't seem relevant to our app, these are regulations for the water treatment, but nothing about our app

Comment (MitchHynes) [reply docs/SRS-Volere/SRS.tex:356]: I was thinking it more so as we can't design our app with any features which would be in violation of these compliance regulations. The existing entry/exit protocols are built around adhering to these regs, and things like the requirement to generate reports come from a regulatory need to adhere to legislation.

File: docs/SRS-Volere/SRS.tex (hunk 1 above)
Comment (MitchHynes) [reply docs/SRS-Volere/SRS.tex]: Thanks, added identifiers

File: docs/SRS-Volere/SRS.tex (hunk 2 above)
Comment (kyle4908) [reply docs/SRS-Volere/SRS.tex]: do the regulations listed here actually have anything to do with the entry/exit protocols?

Comment (MitchHynes) [reply docs/SRS-Volere/SRS.tex]: The entry/exit protocols would exist to adhere to the requirement in those regs to ensure the integrity of the water supply. These are the requirements that stops us from modifying the existing system in place, such as the physical logbook. 

The compliance requirements can be broad, they're just some requirement boundary that can't be crossed. Example from the template which is similar:
image

Comment (kyle4908) [reply docs/SRS-Volere/SRS.tex]: Right, but that's not my question my question is if the `Safe Drinking Water Act`, `Ontario Water Resources Act`, `Environmental Protection Act`, or `Clean Water Act` are related to the entry/exit protocols of the city? and if not do they have any place here, since they are not related to our application

**Review by kyle4908 — APPROVED** [review]

**Review by Spitgranger — APPROVED** [review]
